selene-lib/src/lints/test_util.rs

```diff
@@ -96,7 +222,10 @@ pub fn test_lint_config_with_output<
     let output_path = path_base.with_extension(output_extension);
 
     if let Ok(expected) = fs::read_to_string(&output_path) {
-        pretty_assertions::assert_eq!(PrettyString(&expected), PrettyString(stderr));
+        pretty_assertions::assert_eq!(
+            PrettyString(&expected.replace("\r\n", "\n")),
+            PrettyString(&stderr.to_string().replace("\r\n", "\n"))
+        );
```

Comment:
Without normalizing, this [one specific lint](https://github.com/Kampfkarren/selene/actions/runs/6568529163/job/17843206887?pr=561) fails. I'm confused why this wasn't an issue previously and for the others.